modules/effects/src/effects_resolver.ts

```diff
@@ -51,9 +57,9 @@ export function mergeEffects(
   return merge(...observables$);
 }
 
-function resubscribeInCaseOfError<T extends Action>(
+export function resubscribeInCaseOfError<T extends Action>(
```

Comment:
Yep agreed, I've done a little more that that though that needs confirmation. In order to both ensure type safety and describe intent I renamed that function to `defaultEffectsErrorHandler` and made it a constant so that it can be strongly typed with the exported type symbol. wdyt?

Reply:
I agree, the name makes it more clear